src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -464,10 +464,10 @@ export default class Aircraft {
             // an aircraft was given a radial  clearance
             if (this.isHeadingInsideDepartureWindow()) {
                 this.radioCall('switching to center, good day', 'dep');
-                prop.game.score.departure += 1;
+                window.gameController.events_recordNew('DEPARTURE');
```

Comment:
make sure to update these to use the new enumeration.  There are several in this file that are still strings.